src/app/commands/cmd_import_sprite_sheet.cpp

```diff
@@ -251,14 +267,19 @@ class ImportSpriteSheetWindow : public app::gen::ImportSpriteSheet
         sheetType()->setSelectedItemIndex((int)app::SpriteSheetType::Rows-1);
 
       gfx::Rect defBounds = m_docPref->importSpriteSheet.bounds();
-      gfx::Size defPaddingBounds = m_docPref->importSpriteSheet.paddingBounds();
       if (defBounds.isEmpty())
         defBounds = m_docPref->grid.bounds();
       onChangeRectangle(defBounds);
+
+      gfx::Size defPaddingBounds = m_docPref->importSpriteSheet.paddingBounds();
+      if (defPaddingBounds.w < 0 | defPaddingBounds.h < 0)
```

Comment:
Please almost never use `|` to separate conditions (`|` is for [bitwise operations](https://en.wikipedia.org/wiki/Bitwise_operation), use `||` (or `&&`) instead which are [short-circuit evaluations](https://en.wikipedia.org/wiki/Short-circuit_evaluation).
```suggestion
      if (defPaddingBounds.w < 0 || defPaddingBounds.h < 0)
```